### ISSUE TITLE
Homework 06

### ISSUE THREAD

**Issue by mynamedaike** [STAT545-UBC/Discussion#413]
Hello @jennybc,

There is a part called **Work with the candy data** in this homework. I wanna do this part, but the page to which the link **this homework from 2015** is directed seems to be deleted. 

Sincerely,
Ke Dai

**Comment by jennybc** [comment]
Someone reported similar quite a while ago and I fixed the link. Maybe you need to refresh? Or else you need to be more specific, provide (broken) links, etc.

**Comment by mynamedaike** [comment]
Hello @jennybc,

Now it's Ok. Thanks. I have another problem. The character \ cannot be displayed in the md file. How can I fix the problem. Thanks in advance.

Sincerely,
Ke Dai

**Comment by jennybc** [comment]
> The character \ cannot be displayed in the md file. How can I fix the problem

What md file? What character?

**Comment by mynamedaike** [comment]
@jennybc I mean the backslash "\" cannot be displayed in a md file. I write the backslash "\" in a rmd file, and then I knitted it to md file. The backslash cannot be displayed normally.

**Comment by ksedivyhaley** [comment]
I believe you need to escape the backslash. Try adding another backslash
(\\)

**Comment by jennybc** [comment]
Yes what @ksedivyhaley said. This is why the joke "just add another backslash" is funny 😅.

For even more about this, you can read my [shruggie study](http://rpubs.com/jennybc/shruggie) ¯\\\_(ツ)\_/¯
